waku/waku_test.go

```diff
@@ -1519,7 +1567,9 @@ func TestWakuTimeDesyncEnvelopeIgnored(t *testing.T) {
 		require.NoError(t, err)
 	case <-time.After(time.Second):
 	}
-	rw2.Close()
+	if err := rw2.Close(); err != nil {
+		t.Errorf("error closing MsgPipe, '%s'", err)
+	}
```

Comment:
[Error handling is good](https://github.com/status-im/status-go/pull/1957/files#r417328424)